site/gatsby-config.js

```diff
@@ -9,6 +9,13 @@ module.exports = {
   plugins: [
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-sass',
+    {
+      resolve: `gatsby-plugin-styled-components`,
+      options: {
+        // Shows component displaynames in CSS classnames
+        // displayName: true
```

Comment:
Should we just remove this or do we need the comment?

Reply:
I can remove it